packages/ndla-ui/src/Figure/FigureExpandButton.tsx

```diff
@@ -7,26 +7,29 @@
  */
 
 import React from 'react';
-import { ArrowCollapse } from '@ndla/icons/common';
+import { Play } from '@ndla/icons/common';
 import { ExpandTwoArrows } from '@ndla/icons/action';
+import { CursorClick } from '@ndla/icons/action';
 
-export function FigureExpandButton({ messages, typeClass }: Props) {
+export function FigureExpandButton({ messages, typeClass, type }: Props) {
```

Comment:
Nå vises forresten ikke ekspander-knapp på bilder i eksempelet /?path=/story/enkle-komponenter--bilde.

Et forslag for å ikke innføre breaking changes er å gjøre type optional og nåværende knapper som i master som default.